src/test/java/org/jboss/windup/web/selenium/EditProject.java

```diff
@@ -620,7 +620,9 @@ public void deleteApplication(String app) {
 				x++;
 			}
 			catch (NoSuchElementException e) {
-				System.out.println("did not work");
+				System.out.println (new Object() {}.getClass().getName() + ":" +
+						new Object() {}.getClass().getEnclosingMethod().getName());
+				System.out.println("Delete did not work");
```

Comment:
I realise that this is something which existed before your changes, but this code - when it works perfectly - prints out an error message "Delete did not work" then breaks out of the loop, which isn't ideal